tests/test_coin_combination.py

```diff
@@ -1,3 +1,10 @@
 from app.coin_combination import get_coin_combination
 
-# write your tests here
+
+def test_check_right_count_of_coins():
+    assert get_coin_combination(17) == [2, 1, 1, 0]  # here we've checked pennies, nickels and dime
```

Comment:
Don't use comments to explain what tests do. 
It's better to create a separate test for each case with a descriptive name.